src/extension-test/unit/cursor-doc/token-cursor-test.ts

```diff
@@ -76,11 +77,26 @@ describe('Token Cursor', () => {
             expect(cursor.offsetStart).toBe(14);
         });
 
+        xit('Does not move past unbalanced top level form', () => {
+            //TODO: Figure out why this doesn't work
```

Comment:
This is utterly strange. It actually does work in the vscode editor, but not like this. I suspect it might have to do with the ”runs amok@ thing, since also here we have a missing close. But that is just a wild guess.